oss.krtirtho.Spotube.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: oss.krtirtho.Spotube
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+rename-desktop-file: spotube.desktop
+rename-appdata-file: spotube.appdata.xml
+command: spotube
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=~/.local/share/oss.krtirtho.spotube:create
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: spotube
+    buildsystem: simple
+    build-commands:
+      - install -dm755 /app/bin /app/spotube /app/share/appdata /app/share/applications /app/share/icons/hicolor/128x128/apps/
+      - sed -i 's|/usr/bin/spotube|spotube %u|' spotube/spotube.desktop
+      - sed -i 's|/usr/share/icons/spotube/spotube-logo|oss.krtirtho.Spotube|' spotube/spotube.desktop
```

Comment:
you should use the command `destkop-file-edit`.

Reply:
I didn't even know this tool existed😅. That makes the process a lot easier

Reply:
ideally these changes should be done upstream as both seems to be wrong from the get go.

Reply:
I cannot change it currently at least not in this release else it'll break the package in other repositories

Reply:
it's ok if it is in a future release. I was just stating that this should only be temporary.

Reply:
Thanks for letting me know this😀